addons/hr_expense/data/hr_expense_data.xml

```diff
@@ -22,20 +21,23 @@
             <field name="standard_price">0.0</field>
             <field name="type">service</field>
             <field name="uom_id" ref="uom.product_uom_unit"/>
-            <field name="uom_po_id" ref="uom.product_uom_unit"/>
             <field name="default_code">TRANS &amp; ACC</field>
             <field name="can_be_expensed" eval="True"/>
             <field name="sale_ok" eval="False"/>
             <field name="purchase_ok" eval="False"/>
             <field name="categ_id" ref="product.product_category_expenses"/>
             <field name="image_1920" type="base64" file="hr_expense/static/img/transport.svg"/>
         </record>
+        <record id="product_uom_km" model="uom.uom">
+            <field name="name">km</field>
+            <field name="relative_uom_id" ref="uom.product_uom_meter"/>
+            <field name="relative_factor">1000.0</field>
+        </record>
```

Comment:
Feels weird/wrong to add this in with `hr_expense`, but if truly no other module uses it then I guess it's ok...

You'll probably have to fix the `industry`  module data/demo afterwards though